CHANGELOG.md

```diff
@@ -1,12 +1,11 @@
 # Release History
 
-## 1.0.0-preview.6 (Unreleased)
-
-### New Features
+## 1.0.0 (2020-09-21)
 
 ### Breaking Changes
 
-- Update provisioning client struct member name in `az_iot_provisioning_client_register_response` from `registration_result` to `registration_state`.
+- Removed `az_storage_blobs.h`, including APIs related to storage service such as `az_storage_blobs_blob_client_init()` and `az_storage_blobs_blob_upload()`, and types such as `az_storage_blobs_blob_client` and `az_storage_blobs_blob_client_options`.
```

Comment:
Add a comment that the these will ship in the 1.1.0 release.  Also mention they continue to be available as preview from the branch, https://github.com/Azure/azure-sdk-for-c/tree/feature/StorageBlobs